docs/pages/examples/send-transaction.en-US.mdx

```diff
@@ -90,10 +90,8 @@ export function SendTransaction() {
   const [debouncedAmount] = useDebounce(amount, 500)
 
   const { config } = usePrepareSendTransaction({
-    request: {
```

Comment:
Mind fixing the indentation here?

Reply:
Sure, my pleasure

Reply:
Thanks for the notice, I've done fixing the indentation.